core-feature-pack/src/main/resources/content/bin/jboss-cli.ps1

```diff
@@ -10,7 +10,7 @@ $scripts = (Get-ChildItem $MyInvocation.MyCommand.Path).Directory.FullName;
 $SERVER_OPTS = Process-Script-Parameters -Params $ARGS
 
 # Override ibm JRE behavior
-$SERVER_OPTS += "-Dcom.ibm.jsse2.overrideDefaultTLS=true"
+$JAVA_OPTS += "-Dcom.ibm.jsse2.overrideDefaultTLS=true"
```

Comment:
How does that change relates to the `JBOSS_HOME` env?

With that change, `SERVER_OPTS` no longer contains the `com.ibm.jsse2.overrideDefaultTLS` property. Is that intentional?

Reply:
Ah, no, it is related the problem described in the downstream Jira comment: https://issues.jboss.org/browse/JBEAP-14095?focusedCommentId=13517286&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13517286. 

I think that is a mistake introduced by PR: https://github.com/wildfly/wildfly-core/pull/2464/files

I am fine to make a new issue and PR with it if you think it necessary :)

Reply:
The changes in https://github.com/wildfly/wildfly-core/pull/2464/files seems fine.

@gaol could you please just revert this change in this PR so that we can fix the JBOSS_HOME issue? thanks